lib/sizzle.patched.js

```diff
@@ -1488,6 +1491,20 @@ Expr = Sizzle.selectors = {
 
 		"contains": markFunction(function( text ) {
```

Comment:
Why cannot we move `:contains` from Sizzle to our code?

More we add to Sizzle, harder it will be to upgrade it to a newer version.